about.html

```diff
@@ -0,0 +1,179 @@
+        culture, and open knowledge with CC. Free software supporters, Wikipedia
+        activists, open knowledge foundation activists, art galleries,
+        libraries, museums, archives, governments, foundations, lawyers, social
+        activists, etc. various people active in the field of open content and
+        open knowledge gather in one place to share richer and more dynamic We
+        hope that it will be an opportunity to share our thoughts on creating an
+        ecosystem.
+      </p>
+
+      <p class="place">
+        please contact us per email for any further questions about CC Global
+        Summit 2015!
+      </p>
+      <a class="date" href="mailto:skj997788@gmail.com"> Mail US </a>
+    </header>
+    <section>
+      <div class="wrap program-info-wraper">
+        <section>
+          <h2 class="normal">Creative Commons</h2>
+          <h2>Global Summit 2015 Logo</h2>
+          <div class="guide-bar"></div>
+          <div class="program-list">
+            <div class="program-card">
+              <p>
+                The logo of CC Global Summit was decided through the logo
+                competition from 8.jun to 7.july. After online public voting and
+                selection Committee judging process. the logo from Naresh
+                Agrawal won the prize.After consultations from Mr. Agrawal we
+                optimized the prize-wining-logo for thi syears summit as
+                following:
+              </p>
+            </div>
+            <div class="img">
+              <img src="./images/aboutlogo.png" alt="logo image" />
+            </div>
+          </div>
+        </section>
+      </div>
+    </section>
+
+    <section>
+      <div class="wrap program-info-wraper">
+        <section>
+          <h2>See The Past CC Global Summits</h2>
+          <div class="guide-bar"></div>
+          <div class="program-list">
+            <div class="program-card">
+              <p>
+                Take alook at the past two CC Global Summits which took place in
+                Buenos and in Warsaw.
+              </p>
+            </div>
+            <div class="footer-img">
+              <div class="img1">
+                <div class="background-color">
+                  <h3>2013</h3>
+                  <p>CC Global Summit 2013 in Buenos Aires</p>
+                </div>
+              </div>
+              <div class="img2">
+                <div class="background-color">
+                  <h3>2011</h3>
+                  <p>CC Global Summit 2011 in Warsaw</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+      </div>
+    </section>
+
+    <section>
+      <div class="wrap partner-section">
+        <section>
+          <h2>Partner</h2>
+          <div class="guide-bar"></div>
+          <div class="partner-div">
+            <p>
+              <span class="mozilla"> mozilla </span>
+              <span class="google"> Google </span>
```

Comment:
- [ ] Please fix this weriedweirdponsive behavior which caused by  the images width 

![image](https://user-images.githubusercobehaviorntent.com/74744787/163301030-4b7be157-cf5f-448d-b090-673a55e8a6a1.png)

![image](https://user-images.githubusercontent.com/74744787/163301116-ae6f7ef7-3af6-488e-a044-3671d24e1973.png)

![image](https://user-images.githubusercontent.com/74744787/163301171-21d4bf31-eee4-4bf0-8a9b-2c15845bf7da.png)
I have noticed here that you have set the width to auto which is unreasonable 
![image](https://user-images.githubusercontent.com/74744787/163301256-a204ef7f-69c6-45af-9db8-d64b172354ab.png)